docs/configuredgraphfactory.adoc

```diff
@@ -83,7 +83,7 @@ for which you have instantiated _and_ the references are stored inside the `Janu
 
 IMPORTANT: This is an irreversible operation that will delete all graph and index data.
 
-IMPORTANT: To ensure all graph representations are consistent across all JanusGraph nodes in your cluster, remove the graph from the `JanusGraphManager` graph reference tracker on all nodes in your cluster: `ConfiguredGraphFactory.close("graphName");`.
+IMPORTANT: To ensure all graph representations are consistent across all JanusGraph nodes in your cluster, this removes the graph from the `JanusGraphManager` graph cache on every node in the cluster, assuming each node has been properly configured to use the `JanusGraphManager`.
```

Comment:
I think this suggestion makes alot of sense; I prefer to leave the call out and link to the new section about using a multinode JG cluster. I also think the information in this PR [here](https://github.com/JanusGraph/janusgraph/pull/884#issuecomment-368149520) will be relevant, so I suggest creating a _separate_ PR with this new doc section that includes the information relevant to both PRs; this will avoid merge-conflicts between these two PRs.

Reply:
separate PR for docs update is fine, see https://github.com/JanusGraph/janusgraph/issues/938